kaggle_environments/main.py

```diff
@@ -158,24 +177,24 @@ def action_handler(args):
     try:
         if args.action == "list":
             return action_list(args)
-        elif args.action == "http-server":
+        if args.action == "http-server":
```

Comment:
super nit: I find the previous form with `elif`s nicer / stronger, but definitely a personal decision and not important :)